docs/SettingUp.adoc

```diff
@@ -37,7 +37,7 @@ Do not disable them. If you have disabled them, go to `File` > `Settings` > `Plu
 
 == Verifying the setup
 
-. Run the `seedu.address.Main` and try a few commands
+. Run the `seedu.description.Main` and try a few commands
```

Comment:
I think the package name should be renamed to `seedu.algobase.Main` instead of `seedu.description.Main` as `address` in this context is referring to address book.